sumcheck/src/gpu/sumcheck.rs

```diff
@@ -97,34 +111,14 @@ impl<F: PrimeField + FromFieldBinding<F> + ToFieldBinding<F>> GPUApiWrapper<F> {
                 shared_mem_bytes: 0,
             };
             unsafe {
-                sum.launch(
-                    launch_config,
-                    (
-                        &mut *buf,
-                        &mut *round_evals,
-                        size >> 1,
-                        round * (max_degree + 1) + k,
-                    ),
-                )?;
+                sum.launch(launch_config, (&mut *buf, &mut *round_evals, size >> 1, k))
+                    .map_err(|e| LibraryError::Driver(e))?;
             };
         }
-        Ok(())
-    }
-
-    pub(crate) fn squeeze_challenge(
-        &self,
-        round: usize,
-        challenges: &mut CudaViewMut<FieldBinding>,
-    ) -> Result<(), DriverError> {
-        let squeeze_challenge = self.gpu.get_func("sumcheck", "squeeze_challenge").unwrap();
-        let launch_config = LaunchConfig {
-            grid_dim: (1, 1, 1),
-            block_dim: (1, 1, 1),
-            shared_mem_bytes: 0,
-        };
-        unsafe {
-            squeeze_challenge.launch(launch_config, (challenges, round))?;
-        }
+        let fes = self
```

Comment:
How about `round_evals` for clarity?

Reply:
There is `round_evals` for argument, so `round_eval_values` may be okay I think

Reply:
I will merge this PR and if you think it is not appropriate then please let me know